src/smspark/platform_utils.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Utils distinguishs between configuration differences between training and processing."""
+import os
+
+TRAINING_RESOURCE_CONFIG_PATH = "/opt/ml/input/config/resourceconfig.json"
```

Comment:
Lets avoid using TRAINING specific terms here. You can define a list of resourceconfig paths and then check both paths and whichever location you find a resourceconfig, get that one